testing/SimHarness.cpp

```diff
@@ -45,6 +45,12 @@ void SimHarness::RunForMillis(size_t t) {
   }
 }
 
+void SimHarness::DelayForMillis(size_t t) {
```

Comment:
Is there a reason to have this wrapper rather than just calling the regular arduino "delay()" function, which defaults to ms? It looks like it's already there in the sim.
![image](https://user-images.githubusercontent.com/45416/153500490-cad33944-7bd6-43ef-821b-38e11b76da12.png)


..and it looks like using micros may...not be all that useful right now:

![image](https://user-images.githubusercontent.com/45416/153500582-81aac558-ffba-4ad9-904c-597213609820.png)

Reply:
You're absolutely correct.  For some reason, I was thinking that the parameter for `delay()` was in seconds, not milliseconds.  I'll fix it and force-push.

Reply:
Also, I couldn't find the code for the simulator's `millis()` function when I looked for it.

Reply:
Fair :)

(It's in Arduino.cpp in the virtual/ arduino core)